### DISCUSSION TITLE
Update: Nevermind I got it working 🤞

### DISCUSSION THREAD

**Discussion by jserodio** [Rparadise-Team/Koriki#14] [General]

**Comment by eduardofilo** [comment]
In case anyone else is interested, there is this option:
https://github.com/Rparadise-Team/Koriki/wiki/En:-05.-FAQ#q8-is-there-an-sleep-mode-in-koriki

On the other hand, about what you said about avoiding card corruption during forced shutdown by holding Power, that shutdown is controlled, i.e. a poweroff is sent to the system, so there should be no problems with SD corruption.
